test/gui/tst_syncing/test.feature

```diff
@@ -70,6 +70,17 @@ Feature: Syncing files
         And the user selects "ownCloud" as a remote destination folder
         Then the sync all checkbox should be checked
 
+    @issue-9797
+    Scenario: Sync all is selected by default expected behavior
```

Comment:
```suggestion
    Scenario: Sync all is selected by default (expected behavior)
```
I think the `expected behavior` is extra data, demonstrating how the `Sync all is selected by default` should act. I I would suggest to put it inside brackets . 

Or even better if you rephrase the scenario to a unique title.